### PR TITLE
Fix for updater missing sending PRs for few repos

### PR DESCRIPTION
**Pull request by VijayKumarMidde**
Sometimes updater is missing to send PRs to all the repos. To get the list of repositories updater uses `GHMyself.listRepositories()` method that returns a `PagedIterable<GHRepository>` closure. We loop through this `Iterable` to verify if all the forks were created and loop through the same `Iterable` again to send PRs. Sometimes if Git is flaky, `Iterable` fails to return all the repos in the second iteration and updater misses sending PRs to those repos.

### CODE REVIEW

File: dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GitHubUtil.java [hunk 1]
```diff
@@ -180,7 +177,24 @@ public GHContent tryRetrievingContent(GHRepository repo, String path, String bra
                 Thread.sleep(TimeUnit.MINUTES.toMillis(1));
             }
         }
-
         return listOfRepos;
     }
+
+    /**
+     * Returns a <code>java.util.Map</code> of GitHub repositories owned by a user. Returned map contains repository names
```
Comment (justinharringa) [inline dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GitHubUtil.java]: Could perhaps say that the Map's key is the name of the repo.

File: dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GitHubUtil.java [hunk 2]
```diff
@@ -180,7 +177,24 @@ public GHContent tryRetrievingContent(GHRepository repo, String path, String bra
...
+     * mapped to their corresponding GitHub repository objects.
+     * @param user
+     * @return
+     */
+    public Map<String, GHRepository> getReposForUserAtCurrentInstant(GHMyself user) {
+        Map<String, GHRepository> repoByName = new HashMap<>();
+        if (user == null) {
+            return repoByName;
```
Comment (justinharringa) [inline dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GitHubUtil.java]: Might be useful to cover this with a unit test. Was this situation covered by a test for `getGHRepositories()`?

Comment (VijayKumarMidde) [reply dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GitHubUtil.java]: Added unit test for the `null` case.

File: dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GitHubUtil.java (hunk 1 above)
Comment (VijayKumarMidde) [reply dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GitHubUtil.java]: Updated PR

**Pull request merged**